codespell_lib/data/dictionary.txt

```diff
@@ -12494,6 +12499,7 @@ wirh->with
 wirte->write
 wirtes->writes
 wirting->writing
+wit hthe->with the
```

Comment:
I think currently we cannot find errors with spaces in them, can you check?

If we can't, are you up for modifying `test_dictionary_formatting` to ensure no whitespace characters are in `err`?

Reply:
You are right. No way to match word with space, right now.
I remove this. I'm going to add a few commits, so I can empty my LO queue.

Thanks a lot,
Andrea